website/source/guides/operations/reference-architecture.html.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: "guides"
+page_title: "Nomad Reference Architecture"
+sidebar_current: "guides-operations-reference-architecture"
+description: |-
+  This document provides recommended practices and a reference
+  architecture for HashiCorp Nomad production deployments.
+product_version: 0.8
+---
+
+# Nomad Reference Architecture
+
+This document provides recommended practices and a reference architecture for HashiCorp Nomad production deployments. This reference architecture conveys a general architecture that should be adapted to accommodate the specific needs of each implementation.
+
+The following topics are addressed:
+
+- [Reference Architecture](#ra)
+- [Deployment Topology within One Datacenter](#one-dc)
+- [Deployment Topology within Multiple Datacenters](#multi-dc)
+- [Network Connectivity Details](#net)
+- [Deployment System Requirements](#system-reqs)
+- [High Availability](#high-availability)
+- [Failure Scenarios](#failure-scenarios)
+
+This document describes deploying a Nomad cluster in combination with, or with access to, a [Consul cluster](/guides/operations/consul-integration/index.html). We recommend the use of Consul with Nomad to provide automatic clustering, service discovery, health checking and dynamic configuration.
+
+## <a name="ra"></a>Reference Architecture
+
+A Nomad cluster typically comprises three or five servers (but no more than seven) and a number of client agents. A cluster can be deployed in a single physical datacenter or it can span multiple datacenters/regions. There is an important distinction to make in that Nomad divides infrastructure into regions, which are served by one Nomad server cluster, but can manage multiple datacenters. For example, a _US Region_ can include datacenters _us-east-1_ and _us-west-2_.
```

Comment:
Suggest rewording: "A Nomad cluster typically comprises three or five servers (but no more than seven) and a number of client agents. Nomad differs slightly from Consul in that it divides infrastruture into regions which are served by one Nomad server cluster, but can manage multiple datacenters or availability zones. For example, a _US Region_ can include datacenters _us-east-1_ and _us-west-2_."